src/labelle/lib/render_engines/tests/test_render_engines.py

```diff
@@ -93,6 +93,7 @@ def test_barcode_render_engine(request, image_diff):
         (BarcodeType.EAN, "123456789012"),
         (BarcodeType.UPC, "12345678901"),
         (BarcodeType.CODE39, "123"),
+        (BarcodeType.CODE128, ""),
```

Comment:
Hmm, this reminds me: for the last patch release I added a hack where the empty string for code128 gets replaced by a space character to avoid an exception. Is this how we want to handle it? We should probably add another test case here for `(BarcodeType.CODE128, " ")`.